From 8f07a61436c4244d865a2b0d4e11da0bf721ae2e Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Mon, 28 Jun 2010 01:11:01 +0200
Subject: [PATCH] Revert "NFO: After careful measuring we noticed that this index isn't faster"

Causes serious performance regression during crawling stage.
This reverts commit 8f9c0853f0ea69c329057394a1ed411abdad50e6.
---
 data/ontologies/33-nfo.ontology |    3 ++-
 data/ontologies/Indices.list    |    2 ++
 2 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/data/ontologies/33-nfo.ontology b/data/ontologies/33-nfo.ontology
index 6a60098..4e0e4f2 100644
--- a/data/ontologies/33-nfo.ontology
+++ b/data/ontologies/33-nfo.ontology
@@ -12,7 +12,7 @@
 
 nfo: a tracker:Namespace, tracker:Ontology ;
 	tracker:prefix "nfo" ;
-	nao:lastModified "2010-06-17T17:06:00Z" .
+	nao:lastModified "2010-04-30T16:12:00Z" .
 
 nfo:Document a rdfs:Class ;
 	rdfs:label "Document" ;
@@ -330,6 +330,7 @@ nfo:belongsToContainer a rdf:Property ;
 	rdfs:subPropertyOf nie:isPartOf ;
 	nrl:maxCardinality 1 ;
 	rdfs:domain nie:DataObject ;
+	tracker:indexed true ;
 	rdfs:range nfo:DataContainer .
 
 nfo:aspectRatio a rdf:Property ;
